ddl/ddl.go

```diff
@@ -506,6 +508,47 @@ func (d *ddl) addColumn(schema model.CIStr, tbl table.Table, spec *AlterSpecific
 	return errors.Trace(err)
 }
 
+func updateDefaultValue(ctx context.Context, t *tables.Table, col *column.Col) error {
```

Comment:
Any comment for this function? Or pick up a better name. "updateDefaultValue" is a little confused. "updateOldRows" may be better.